e2e_test/batch/basic/make_time.slt.part

```diff
@@ -9,20 +9,31 @@ SELECT make_timestamptz(1973, 07, 15, 08, 15, 55.33);
 query T
 SELECT make_timestamptz(-1973, 07, 15, 08, 15, 55.33);
 ----
--1973-07-15 08:15:55.330+00:00
+-1972-07-15 08:15:55.330+00:00
 
-query error Invalid parameter sec: invalid sec
+query error Invalid parameter year, month, day: invalid date: -3-2-29
```

Comment:
In postgres we have `date field value out of range: -4-02-29`. Note the `-4` here. Are we behaving correctly?

Reply:
```
> SELECT make_date(-4, 02, 29);
ERROR:  date field value out of range: -3-02-29
> SELECT make_timestamp(-4, 02, 29, 08, 15, 55.33);
ERROR:  date field value out of range: -4-02-29
```
Pg behaves differently on `make_date` and `make_timestamp`. Seems to be a misaligned design. So I think it's OK if we don't follow it strictly.